prowler/providers/aws/services/s3/s3_service.py

```diff
@@ -381,14 +380,14 @@ def __get_bucket_tagging__(self, bucket):
                 )
 
 
-################## S3Control
 class S3Control(AWSService):
     def __init__(self, provider):
         # Call AWSService's __init__
         super().__init__(__class__.__name__, provider, global_service=True)
-        self.account_public_access_block = self.__get_public_access_block__()
+        self.account_public_access_block = self._get_public_access_block()
+        self.access_points = self._list_access_points()
```

Comment:
You need to iterate `list_access_points` through all the regions since the access points are linked to a region and the account, they are not global.
You can do it by using `self.__threading_call__(self._list_access_points)`